pageserve/templates/events.html

```diff
@@ -36,4 +36,28 @@ <h1>{% block title %}Sub-Events{% endblock title %}</h1>
 {% endfor %}
 
 <a href='/v1/'>See all posts</a>
-{% endblock content%}
\ No newline at end of file
+
+<h2>Create New Event</h2>
```

Comment:
Do you think we might want event adding on a separate page from showing all the events?

Reply:
Single-page app seems reasonable, if the list extends as you add a new event.

Reply:
I'm going to leave it as single-page for now